src/webots/nodes/utils/WbUrl.cpp

```diff
@@ -302,3 +218,18 @@ const QString WbUrl::computeLocalAssetUrl(const WbNode *node, QString url) {
     // when streaming a release (or nightly), "webots://" urls must be inferred
     return WbUrl::computePath(node, "url", url, false);
 }
+
+const QString WbUrl::computePrefix(const QString &rawUrl) {
+  QString url = rawUrl;
+  if (url.startsWith(WbNetwork::instance()->cacheDirectory()))
+    url = WbNetwork::instance()->getUrlFromEphemeralCache(url);
+
+  if (isWeb(url)) {
+    QRegularExpression re("(https://raw.githubusercontent.com/cyberbotics/webots/[a-zA-Z0-9\\_\\-\\+]+/)");
```

Comment:
This is a general comment and doesn't have to be addressed in this branch.
However this regular expression is repeated 4 times in the code and the rawgit URL is hard coded about 10 times.
(same for the `webots://`).
This is not very good and hardcoded  text and common manipulations of URLs should be defined in a single place (for example here in WbUrl class).